editor/components/toolbar/style.scss

```diff
@@ -29,6 +30,17 @@
 		background-color: $dark-gray-500;
 		color: $white;
 	}
+
+	/* Optional number, e.g. for headings */
+	span {
```

Comment:
In an effort to avoid element selectors, might be wise to add a class to the styled span. It's very unclear we're targeting additional text of the `IconButton`.